src/librustdoc/visit.rs

```diff
@@ -1,5 +1,11 @@
 use crate::clean::*;
 
+/// allows a type to traverse the cleaned ast of a crate.
+///
+/// note that like [`rustc_ast::visit::Visitor`], but
+/// unlike [`rustc_lint::EarlyLintPass`], if you override a
+/// `visit_*` method, you will need to manually recurse into
+/// its contents.
```

Comment:
`content` is never plural iirc

Reply:
"contents" is definitely a word, it implies there is likely to be more than one thing contained.

eg. table of contents

Reply:
https://dictionary.cambridge.org/fr/grammaire/grammaire-britannique/content-or-contents

Seems tricky. So I think either way is fine.